### PR TITLE
Added possibility for dynamically generate RSS feeds from Telegram channels

### PR DESCRIPTION
**Pull request by WeaxMe**
Added possibility for dynamically generate RSS feeds from Telegram channels.

### Setup
1. Install dependencies `pip install -r requirements.txt`
2. Run script `setup_telegram.py` - this will create Telegram session file. You must enter your phone and then code which you will receive in Telegram
3. Run migration

### Things for think
Need decide how handle connection to Telegram, because right now it connects to Telegram on each request and then close connection.
Need test and need decide what to do with Telegram connection:
* Create connection pool
* Create singe connection to Telegram and use queue for load messages

Right now I don't know exactly how Telethon client works, so I can't decide how effectively implement connection now. I need time for research.

### CODE REVIEW

**Comment by WeaxMe** [comment]
Hello!
@vas3k I added changes which we discussed. Please review changes

File: infomate/urls.py [hunk 1]
```diff
@@ -15,4 +16,5 @@
 
     path("<slug:board_slug>/", board, name="board"),
     path("<slug:board_slug>/export/", export, name="export"),
+    path("rss/telegram/<str:channel>", TelegramChannelFeed(), name="telegram_channel_feed")
```
Comment (vas3k) [inline infomate/urls.py]: Let's rename it to `parsing/telegram/<str:channel>/`. Just a matter of taste :)

File: parsing/utils.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
+from enum import Enum
+
+
+class TelegramChannel:
```
Comment (vas3k) [inline parsing/utils.py]: Everybody loves `utils.py` :D
But let's give this file some meaningful name, keeping in mind that in the future other parsers will be added here besides Telegram. Maybe let's group them into folder, idk.

File: setup_telegram.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import os
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "infomate.settings")
+import django
+django.setup()
+
+from telethon.sync import TelegramClient
+from django.conf import settings
+
+
+with TelegramClient(settings.TELEGRAM_SESSION_FILE, settings.TELEGRAM_APP_ID, settings.TELEGRAM_APP_HASH) as client:
```
Comment (vas3k) [inline setup_telegram.py]: Could you explain please why do we need this file and what does it do?
I can't see any scripts calling it.

Comment (WeaxMe) [reply setup_telegram.py]: This file need for initialize Telegram session. For initialize Telegram session you must run this script manually. This need only first time, before launch application. 
1. You enter your phone number
2. You receive code in your Telegram account and send that code in console for initialize Telegram session

**Comment by WeaxMe** [comment]
I added changes.
@vas3k  please review

**Pull request merged**